tabs/pid_tuning.css

```diff
@@ -287,7 +302,7 @@
 }
 
 .tab-pid_tuning table td {
-    padding: 1px;
+    padding: 5px;
```

Comment:
I can't reproduce any of this. What platform are you testing with?

Also, do you attribute the problems you are getting to the change to https://github.com/betaflight/betaflight-configurator/pull/272/files/efdf41d1493d22cc7abbb06d8e83ea7bba01ed9f#diff-d7b40b26e0f2a6ff1a2e9ee41a84f491L289? Why?